app/javascript/components/patient/assessment/actions/SymptomOnset.js

```diff
@@ -15,43 +16,37 @@ class SymptomOnset extends React.Component {
     super(props);
     this.state = {
       symptom_onset: this.props.patient.symptom_onset,
+      user_defined_symptom_onset: this.props.patient.user_defined_symptom_onset,
+      showSymptomOnsetModal: false,
       loading: false,
     };
+    this.origState = Object.assign({}, this.state);
   }
 
-  handleDateChange = date => {
-    this.setState({ symptom_onset: date }, () => {
-      if (date && this.props.patient.user_defined_symptom_onset) {
-        this.handleSubmit('Are you sure you want to manually update the symptom onset date?');
-      } else if (date && !this.props.patient.user_defined_symptom_onset) {
-        this.handleSubmit(
-          'Are you sure you want to manually update the symptom onset date? Doing so will result in the symptom onset date no longer being auto-populated by the system.'
-        );
-      } else {
-        this.handleSubmit(
-          'Are you sure you want to clear the symptom onset date? Doing so will result in the symptom onset date being auto-populated by the system.'
-        );
-      }
-    });
-  };
-
-  handleSubmit = async confirmText => {
-    if (await confirmDialog(confirmText)) {
-      this.submit();
-    } else {
-      this.setState({ symptom_onset: this.props.patient.symptom_onset });
+  openSymptomOnsetModal = date => {
```

Comment:
any reason why this can't just be `openModal`?

Reply:
can you also move this to right above the close modal for consistency

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-c70b6a432860cf1cff1aa3bc624a2b72da37531257c62855dba8ee265610f6e9R49